orchestrator/modules/rag/retrieval_filters.py

```
@@ -0,0 +1,183 @@
+    if ws is None and require_workspace:
+        raise RetrievalScopeError(
+            "Retrieval scope has no workspace_id — refusing to search unscoped (fail-closed)."
+        )
+
+    raw_terms: List[str] = []
+    if teams:
+        raw_terms = list(teams)
+    elif team is not None:
+        raw_terms = [team]
+    elif agent is not None and getattr(agent, "team", None):
+        raw_terms = [getattr(agent, "team")]
```

Ruff (0.15.15)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)